tests/basic-end-to-end-tests.bats

```diff
@@ -22,6 +22,7 @@ setup_file() {
 
 @test  "[Basic end-to-end tests] Update privileged pod policy to check only UPDATE operations" {
         run kubectl --context $CLUSTER_CONTEXT patch clusteradmissionpolicy privileged-pods --type=json -p ' [{ "op": "remove", "path": "/spec/rules/0/operations/1" }, { "op": "replace", "path": "/spec/rules/0/operations/0", "value": "UPDATE" } ]'
+        run kubectl --context $CLUSTER_CONTEXT wait --timeout $TIMEOUT --for=condition=PolicyActive clusteradmissionpolicies --all
```

Comment:
After a patch, waiting for the `PolicyActive` condition is not that relevant. We don't transition the `PolicyActive` back from `"True"` to `"False"`. Once the webhook has been installed the first time, `PolicyActive` condition will be always `"True"`, so I would not add this wait here, given it can be misleading.

Reply:
Or, is this fixing something we were missing?

Reply:
oh yes, you are right! We set the status to `pending`, but the `PolicyActive` condition does not change.
Sometimes (not frequently) I get an error [here](https://github.com/kubewarden/kubewarden-end-to-end-tests/blob/main/tests/basic-end-to-end-tests.bats#L29), I think it was because the webhook was not updated from the previous step yet

Reply:
I run the test several times now and it passed.. I removed the condition since it is not relevant as you mentioned. 
Not sure if there is a race condition. Tests seems to be failing quite often https://github.com/kubewarden/kubewarden-controller/actions/workflows/e2e-tests.yml we should check that in a separate issue

Reply:
I believe the test are failing due something wrong in the runner. I'll merge this because it seems fine and fix the test execution issues in our CI for another PR/issue.